pkg/resource/deploy/deployment_executor.go

```diff
@@ -262,7 +262,7 @@ func (ex *deploymentExecutor) Execute(callerCtx context.Context, opts Options, p
 
 				if event.Result != nil {
 					if !event.Result.IsBail() {
-						ex.reportError("", event.Result.Error())
+						logging.V(3).Infof("deploymentExecutor.Execute(...): error running Program: %v", event.Result.Error())
```

Comment:
the previous would throw a Diagnostic Error message and clutter the display. We have moved the responsibility of displaying program errors into the SDK and have left a logging message in its place.